src/pages/Index.vue

```diff
@@ -2,18 +2,120 @@
   <Layout>
 
     <!-- Learn how to use images here: https://gridsome.org/docs/images -->
-    <h1>Welcome to the Galaxy Mentoring Network</h1>
+    
+    <header class="header">
+    </header>
+      <body>
+        <b-container class="index-container">
+        <b-row>
+          <b-col>
+        <h1>Mentorship...for you!</h1>
 
-    <p>Mentorship...for you!</p>
+        <p>An all in one program targeted to speed up your growth!</p>
+        <a href="https://forms.gle/482EoBYtJozWpYCf9" class="btn" role="button" target="_blank">Apply Now</a>
+            </b-col>
+            <b-col>
+              <img src="/images/illustration.png" alt="two people sitting in front of a computer">
+          </b-col>
+          </b-row>
+        </b-container>
+    
+    <b-container class="index-container">
+     <b-card-group deck>
+         <b-card class="index-card"
```

Comment:
I am not sure if cards are really needed there. Maybe you could use just columns with background colors. Currently it feels like nothing is really aligned:

![image](https://user-images.githubusercontent.com/1842467/153602834-1ada7e95-322d-45b3-b6f0-911e3aad9eae.png)

(I know it will change once we have images)

Reply:
Yeah...and probably in one container, to address the alignment.....